lib/rubocop/cop/style/hash_syntax.rb

```diff
@@ -96,7 +96,13 @@ def word_symbol_pair?(pair)
           return false unless key.sym_type?
 
           sym_name = key.loc.expression.source
-          sym_name !~ /\A:["']|=\z/
+          valid_19_syntax_symbol?(sym_name)
+        end
+
+        def valid_19_syntax_symbol?(sym_name)
+          return false if sym_name =~ /\A:["']/
```

Comment:
@bbatsov makes sense. It looks like currently all of the tagging is to either filter out flavor of ruby or run on any version that matches. I don't currently see a way to say run on anything below 2.2. I can add something like that into the spec_helper if you think that's the correct solution.